compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/interop/InteropValidator.java

```diff
@@ -70,6 +70,7 @@ public static InteropValidator getInstance(CompilerContext context) {
     }
 
     public void validate(ModuleId moduleId, CompilerBackend compilerBackend, BLangPackage bLangPackage) {
+        dlog.setCurrentPackageId(bLangPackage.packageID);
```

Comment:
Shall we add a test for this or improve existing test?

Reply:
Created #31539 to track writing test cases